source/dub/generators/build.d

```diff
@@ -546,7 +546,12 @@ class BuildGenerator : ProjectGenerator {
 				auto result = prg_pid.wait();
 				settings.targetExitStatus = result;
 				runPostRunCommands(m_project.rootPackage, m_project, settings, buildsettings);
-				enforce(result == 0, "Program exited with code "~to!string(result));
+				string error() {
+					string error = "Program exited with code "~to!string(result);
+					version (Posix) error ~= " (" ~ signalToName(-result) ~ ")";
+					return error;
```

Comment:
`core.sys.posix.string.strsignal` should probably also be added to the output, as that gives a text explanation of the signal.
https://linux.die.net/man/3/strsignal